x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ServiceAccountAuthenticator.java

```diff
@@ -50,6 +80,14 @@ public void authenticate(Context context, ActionListener<AuthenticationResult<Au
             return;
         }
         final ServiceAccountToken serviceAccountToken = (ServiceAccountToken) authenticationToken;
+        doAuthenticate(context, serviceAccountToken, wrapForAuthc(authenticationMetrics, serviceAccountToken, listener));
```

Comment:
nit: can you use `InstrumentedSecurityActionListener.wrapForAuthc` instead of the static import to match the other usage (or update the other usage)